packages/commonwealth/client/scripts/views/components/react_quill_editor/QuillLinkModal.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { CWTextInput } from '../component_kit/cw_text_input';
+import { CWButton } from '../component_kit/new_designs/CWButton';
+import {
+  CWModal,
+  CWModalBody,
+  CWModalFooter,
+  CWModalHeader,
+} from '../component_kit/new_designs/CWModal';
+
+type LinkModalProps = {
+  linkText: string;
+  linkUrl: string;
+  setIsModalOpen: (isOpen: boolean) => void;
+  setLinkText: (text: string) => void;
+  setLinkUrl: (url: string) => void;
+  handleAddLink: () => void;
+  isModalOpen: boolean;
+};
+
+export const AddLinkModal = ({
+  linkUrl,
+  linkText,
+  isModalOpen,
+  setIsModalOpen,
+  setLinkText,
+  setLinkUrl,
+  handleAddLink,
+}: LinkModalProps) => {
+  const handleModalClose = () => {
+    setIsModalOpen(!isModalOpen);
+    setLinkText('');
+    setLinkUrl('');
+  };
+  return (
+    <CWModal
+      size="small"
+      visibleOverflow
+      content={
+        <>
+          <CWModalHeader label="Add Link" onModalClose={handleModalClose} />
+          <CWModalBody>
+            <CWTextInput
+              label="Link Text"
+              placeholder="Enter Link Text"
```

Comment:
Please make this placeholder text sentence case, that is the design convention we are using across the platform.

Reply:
i have made the placeholder text sentence case